barretenberg/cpp/src/barretenberg/stdlib/primitives/group/cycle_group.cpp

```diff
@@ -845,23 +835,24 @@ typename cycle_group<Builder>::batch_mul_internal_output cycle_group<Builder>::_
         }
     }
 
-    // Normalize the computed witness points and convert into AffineElement type
+    // Normalize the computed witness points and convert them into AffineElements
     Element::batch_normalize(&operation_transcript[0], operation_transcript.size());
-
     std::vector<AffineElement> operation_hints;
     operation_hints.reserve(operation_transcript.size());
-    for (auto& element : operation_transcript) {
-        operation_hints.emplace_back(AffineElement(element.x, element.y));
+    for (const Element& element : operation_transcript) {
+        operation_hints.emplace_back(element.x, element.y);
     }
 
+    // Construct an in-circuit straus lookup table for each point
     std::vector<straus_lookup_table> point_tables;
     const size_t hints_per_table = (1ULL << TABLE_BITS) - 1;
     OriginTag tag{};
     for (size_t i = 0; i < num_points; ++i) {
-        std::span<AffineElement> table_hints(&operation_hints[i * hints_per_table], hints_per_table);
         // Merge tags
         tag = OriginTag(tag, scalars[i].get_origin_tag(), base_points[i].get_origin_tag());
-        point_tables.emplace_back(straus_lookup_table(context, base_points[i], offset_generators[i + 1], TABLE_BITS));
```

Comment:
Note a bug here where we were not even using the hints constructed a few lines above when we go to construct the `straus_lookup_table`! Corrected on the right.